app/models/assignment.rb

```diff
@@ -267,7 +267,7 @@ def display_for_note
 
   # Returns the maximum possible mark for a particular assignment
   def max_mark(user_visibility = :ta_visible)
-    criteria.where(user_visibility => true, bonus: false).sum(:max_mark).round(2)
+    Float(criteria.where(user_visibility => true, bonus: false).sum(:max_mark).round(2))
```

Comment:
@WeissSa Okay this is interesting! The `.sum` here is returning a `BigDecimal`, which gets represented as a string when encoded into JSON. Since we're rounding to two decimal places I'm okay returning a `Float` here like you're doing, but please explain in the method docstring that we're converting the `BigDecimal` to a `Float`.

Reply:
Will do 🖖